data/generalhw_scripts/rpi_pico_w_keyboard/rpi_pico_w_keyboard.ino

```diff
@@ -0,0 +1,200 @@
+#include <WiFi.h>
+#include <WiFiClient.h>
+#include <WebServer.h>
+
+#include <Keyboard.h>
+#include "keymap.h"
+
+
+#ifndef STASSID
+#define STASSID "openQA-worker"
+#define STAPSK "CHANGE THIS DUMMY TO YOUR PASSWORD"
+#endif
+
+const char *ssid = STASSID;
+const char *password = STAPSK;
+WebServer server(80);
+
+
+// Ringbuffer for the last 10 keyboard inputs to be shown on the webui
+#define LOGBUF_LEN 10
+char* logbuf[LOGBUF_LEN] = {NULL};
+uint8_t logbuf_next_idx = 0;
+
+void logbuf_add(const char* logline) {
+  if (logbuf[logbuf_next_idx] != NULL) {
+    free(logbuf[logbuf_next_idx]);
+  }
+  //Serial.println("LOG:");
+  //Serial.println(logline);
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details